website_sale_resource_booking/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Jairo Llopis <jairo.llopis@tecnativa.com> (https://www.tecnativa.com/)
```

Comment:
Please use the double line format with Tecnativa and url on first line and your name without email address on the second.